src/agent/gs_agent/wrappers/teleop_wrapper.py

```diff
@@ -532,3 +607,159 @@ def close(self) -> None:
     def render(self) -> None:
         """Render the environment."""
         pass
+
+    # Trajectory Recording and Replay Methods
+
+    def start_recording(self) -> None:
+        """Start recording trajectory data."""
+        if self.recording:
+            print("⚠️  Already recording trajectory!")
+            return
+
+        self.recording = True
+        self.trajectory_data = []
+        self.recording_start_time = time.time()
+        print("🔴 Started recording trajectory...")
+        print("   Press 'r' again to stop recording and save.")
+
+    def stop_recording(self) -> None:
+        """Stop recording and save trajectory data."""
+        if not self.recording:
+            print("⚠️  Not currently recording!")
+            return
+
+        self.recording = False
+        recording_duration = time.time() - (self.recording_start_time or 0)
+
+        print(f"🔴 Stopping recording... data_len={len(self.trajectory_data)}")
+
+        if not self.trajectory_data:
+            print("⚠️  No trajectory data recorded!")
+            return
+
+        # Save trajectory to file
+        filename = self._save_trajectory()
+        print("✅ Stopped recording trajectory!")
+        print(f"   Duration: {recording_duration:.2f} seconds")
+        print(f"   Steps recorded: {len(self.trajectory_data)}")
+        print(f"   Saved to: {filename}")
+
+        # Clear trajectory data
+        self.trajectory_data = []
+        self.recording_start_time = None
+
+    def _save_trajectory(self) -> str:
+        """Save trajectory data to file."""
+        # Create trajectories directory if it doesn't exist
+        os.makedirs(TRAJECTORY_DIR, exist_ok=True)
+
+        # Generate filename with timestamp
+        timestamp = int(time.time())
+        filename = f"{TRAJECTORY_FILENAME_PREFIX}{timestamp}{TRAJECTORY_FILE_EXTENSION}"
+        filepath = os.path.join(TRAJECTORY_DIR, filename)
+
+        # Save trajectory data
+        with open(filepath, "wb") as f:
+            pickle.dump(self.trajectory_data, f)
+
+        return filepath
+
+    def _load_latest_trajectory(self) -> list[TrajectoryStep] | None:
```

Comment:
Can you change this function to _load_trajectory, which will take the path or name of the trajectory as an argument?
replay_latest_trajectory will find the name of the latest trajectory and pass it to _load_trajectory.
Also, the example file can take the path as an argument from the user: if no argument is provided, it loads the latest trajectory; otherwise, it loads the one specified by the user.